thriftbp/server.go

```diff
@@ -69,19 +100,27 @@ func NewServer(
 //
 // The TProcessor underlying the server will be wrapped in the default
 // Baseplate Middleware and any additional middleware passed in.
+//
+// The suppressor arg will be used as
+// DefaultProcessorMiddlewaresArgs.ErrorSpanSuppressor,
+// please refer to its documentation for more details.
```

Comment:
It might be good to add that the `Addr` and `Timeout` will be pulled from `bp.Config()`